docs/user-guide/cli-reference/commands/index.md

```diff
@@ -0,0 +1,52 @@
+# CLI Reference
+
+## PANDO Server
+
+### $ ./pando-server help
+Help information for all the commands and their options
+
+### $ ./pando-server init
+Initialize PANDO server and create its configuration file
+
+### $ ./pando-server daemon
+Start PANDO server daemon process
+
+| Flag | Type | Description | Default Value |
+| --- | --- | --- | --- |
+| --account-level | ints | Rank the accounts then set rate limits for them. | [1,10,100,500] |
+| --admin-listen-addr | string | Admin server listen address(in multiaddress format, like /ip4/127.0.0.1/tcp/8999). | | /ip4/127.0.0.1/tcp/8999 |
+| --backup-apikey | string | Estuary api key used to backup metadata files. | |
+| --backup-check-estuary-interval | string | Interval for Pando to check backup deal status in estuary. | 4h0m0s |
+| --backup-estuary-gateway | string | Estuary gateway address used to backup metadata files. | https://api.estuary.tech |
+| --backup-estuary-interval | string | Interval for Pando to back up car files to estuary. | 24h0m0s |
+| --backup-gencar-interval | string | Interval for Pando to generate car files for providers' meta data. | 1m0s |
+| --backup-shuttle-gateway | string | Estuary shuttle gateway address used to backup metadata files. | https://shuttle-4.estuary.tech |
+| --cachestore-dir | string | Directory of cachestore files. | cachestore |
+|  -f, --config-file | string | Load server configuration from a yaml file rather than line flags. | config.yaml |
+| --datastore-dir | string | Directory of datastore files. | datastore |
+| --datastore-type | string | Datastore type, support levelds only for now. | levelds |
+| -p, --disable-speedtest | | Init server configuration without Internet connection speed test. | |
+| --discovery-lotus-gateway | string | Lotus gateway address. | https://api.chain.love |
+| --discovery-policy-allow | | Discovery allow all origin source host. | true |
+| --discovery-policy-trust | | Enable discovery white-list. | true |
+| --external-ip | string | Pando public IP address for API serve | unknown |
```

Comment:
in fact, it is displayed in PandoInfo API's IP,  you can input wrong ip
func (c *Controller) PandoInfo() (*model.PandoInfo, error) {
	ipReplacer := func(multiAddress string, replaceIP string) string {
		splitAddress := strings.Split(multiAddress, "/")
		splitAddress[2] = replaceIP
		return strings.Join(splitAddress, "/")
	}

	return &model.PandoInfo{
		PeerID: c.Core.LegsCore.Host.ID().String(),
		Addresses: model.APIAddresses{
			HttpAPI:      ipReplacer(c.Options.ServerAddress.HttpAPIListenAddress, c.Options.ServerAddress.ExternalIP),
			GraphQLAPI:   ipReplacer(c.Options.ServerAddress.GraphqlListenAddress, c.Options.ServerAddress.ExternalIP),
			GraphSyncAPI: ipReplacer(c.Options.ServerAddress.P2PAddress, c.Options.ServerAddress.ExternalIP),
		},
	}, nil
}